src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -176,32 +178,41 @@ export const Sidebar: React.FC<Props> = ({
     }
   }, [redeemCode, campaignData, refetchAll])
 
+  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
   const discountText = getDiscountText(textKeys)(
     redeemedCampaigns,
     offerData.cost.monthlyGross.currency,
   )
+  const clearDiscountCode = () => {
+    removeDiscountCode()
+      .then(() => {
+        const cookieStorage = new CookieStorage()
+        cookieStorage.setItem('_hvcode', '', {
+          path: '/',
+        })
+      })
+      .then(() => refetchAll())
+  }
+  const hasMultipleQuotes = offerData.quotes.length > 1
 
-  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
+  const hasRedeemedCampaigns = redeemedCampaigns.length > 0
 
   return (
     <>
       <ReactVisibilitySensor partialVisibility onChange={setIsSidebarVisible}>
         {() => (
           <Wrapper>
             <Container>
-              <HeaderTop>
-                <HedvigSymbol />
-                <DiscountInfo>
-                  {isNorwegianBundle && (
-                    <Badge>{textKeys.SIDEBAR_NO_BUNDLE_DISCOUNT_TEXT()}</Badge>
-                  )}
-                  {discountText && <Badge>{discountText}</Badge>}
-                </DiscountInfo>
-              </HeaderTop>
-
+              <DiscountInfo>
```

Comment:
Does it matter for the layout that this container is present even if there are no discounts to show?